cookbook/scripts/archive-all-example.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -e
+
+for row in $(cat flytetester.json | jq -c '.[]'); do
+  echo "tar -zcvf ./release-snacks/flytesnacks-$(echo ${row} | jq -r '.name').tar.gz  ./$(echo ${row} | jq -r '.path')/_pb_output"
+done
```

Comment:
I think you want the flytetester.json file itself to be present in the tar. or at least be present in the release... right?